components/tikv_util/src/logger/mod.rs

```diff
@@ -628,6 +631,14 @@ fn write_log_fields(
     Ok(())
 }
 
+fn get_values() -> OwnedKV<impl SendSyncRefUnwindSafeKV> {
+    slog_o!(
+        "thread_id" => FnValue(|_| {
+            format!("{:#0x}", std::thread::current().id().as_u64())
```

Comment:
the id is fetched every time?

Reply:
That's my understanding, yes. I don't see a way to avoid it other than caching it in a thread local, which is probably not worth it. Another option would be to make this feature optional through a config.

Reply:
Seems it's cheap https://stackoverflow.com/questions/15007771/is-operation-of-getting-id-of-current-thread-time-expensive

Reply:
Inside that implementation std::thread::current().id(), the thread id must be cached inside thread local storage.